src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2667,4 +2667,142 @@
   <data name="ExitCompactOverlayDescription" xml:space="preserve">
     <value>Exit compact overlay</value>
   </data>
+  <data name="Key.Menu" xml:space="preserve">
+    <value>Menu</value>
```

Comment:
Can you add a description to the hotkeys? The same description can be used for all of them.

Reply:
It's done.